File: 03. Ekspresi dan Masukan atau Keluaran/D. Bebek untuk Teman/d_bebek_untuk_teman.go
package main

import (
	"fmt"
)

func main() {
	var a, b int
	fmt.Scanf("%d %d", &a, &b)
	fmt.Printf("masing-masing %d \n", a/b)
	fmt.Printf("bersisa %d \n", a%b)
}

File: 03. Ekspresi dan Masukan atau Keluaran/D. Bebek untuk Teman/d_bebek_untuk_teman.java
import java.util.Scanner;

class d_bebek_untuk_teman {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int A = scan.nextInt();
        int B = scan.nextInt();
        scan.close();
        System.out.print("masing-masing ");
        System.out.println(A / B);
        System.out.print("bersisa ");
        System.out.println(A % B);
    }
}
